Observations/obs.md

```diff
@@ -0,0 +1,152 @@
+    *order* | `String` | the order of an Obs.
+    *encounter* | `String` | what obs are collected and grouped together into. An encounter is a visit.
+    *accessionNumber* | `String` | a unique identifier assigned to each Obs.
+    *groupMembers* | `Array[]: Obs` |  a list of Obs grouped under this Obs
+    *comment* | `String` | the comment written in an Obs.
+    *value* | `String` | various values saved from a given Obs.
+    *status* | ENUM | the status of the observation
+    *interpretation* | ENUM | the enumerations related to the current state of being of an Obs
+    *voided* | `Boolean` | whether the obs is voided or not
+    
+   
+    ```console
+    POST /obs
+    {
+      "person": {
+        "uuid": "070f0120-0283-4858-885d-a20d967729cf",
+        "display": "1001MH - John Smith",
+        "links": [
+          {
+            "rel": "self",
+            "uri": "http://demo.openmrs.org/openmrs/ws/rest/v1/patient/070f0120-0283-4858-885d-a20d967729cf"
+          }
+        ]
+      },
+      "obsDatetime": "2016-11-10T07:37:31.000+0000",
+      "voided": false,
+      "status": "PRELIMINARY",
+      "interpretation": "NORMAL"
+    }
+    ```
+### Update an observation
+
+*  Update a target obs, this method only modifies properties in the request. Returns `404 Not Found` status if the observation does not exist. 
+If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
+status is returned.
+    
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *person* | `Person UUID` | the Person this Obs is acting on.
+    *obsDateTime* | `String` | the time this Obs took place.
+    *concept* | `String` | the coded value/name given to an obs when it is made.
+    *location* | `String` | the location this Obs took place (was taken).
+    *order* | `String` | the order of an Obs.
+    *encounter* | `String` | what obs are collected and grouped together into. An encounter is a visit.
+    *accessionNumber* | `String` | a unique identifier assigned to each Obs.
+    *groupMembers* | `Array[]: Obs` |  a list of Obs grouped under this Obs
+    *comment* | `String` | the comment written in an Obs.
+    *value* | `String` | various values saved from a given Obs.
+    *status* | ENUM | the status of the observation
+    *interpretation* | ENUM | the enumerations related to the current state of being of an Obs
+    *voided* | `Boolean` | whether the obs is voided or not
+   
+    ```console
+    POST /obs
+    {
+      "person": {
+      "uuid": "070f0120-0283-4858-885d-a20d967729cf",
+      "display": "1001MH - John Smith",
+      "links": [
+        {
+          "rel": "self",
+          "uri": "http://demo.openmrs.org/openmrs/ws/rest/v1/patient/070f0120-0283-4858-885d-a20d967729cf"
+        }
+      ]
+      },
+      "obsDatetime": "2016-11-10T07:37:31.000+0000",
+      "voided": true,
+      "status": "PRELIMINARY",
+      "interpretation": "ABNORMAL"
+    }
```

Comment:
If you are updating an existing obs (not creating a new one), then you should be POSTing to the obs resource being updated and not just to `/obs`.  For example, in the obs creation example I gave above, the response included links to the obs that was created, which in my case was `/obs/8ea2e81e-e83b-470a-b463-a79f4181d023`. So, you could use this as an example for updating the observation:

```
POST /obs/8ea2e81e-e83b-470a-b463-a79f4181d023
{
  "value": 71
}
```

or

```
POST /obs/:uuid_of_obs_to_be_updated
{
  "value": 71
}
```